### DISCUSSION TITLE
16 bit color?

### DISCUSSION THREAD

**Discussion by builderdev212** [Bodmer/TFT_eSPI#2206] [Q&A - General]
A bit of a curiosity, is color support based on the display/driver? I have an esp32 and am using it to drive an ili9431 based display. I've always had issues with color accuracy, though I had always assumed it was the display. Is there a certain configuration that needs to be set up in order to support 16-bit color? Or is it a display issue?

**Comment by Bodmer** [comment]
The library uses 16 bit colour which is compatible with the ILI9341. The colour rendition for displays is dependant on a number of factors,  the viewing angle,  the initialisation control register values (gamma etc) and the display manufacturer. This is complicated by Chinese clones of some controllers that show different colour tones.
